### PR TITLE
Adds an EAMXX fully coupled compset

### PR DESCRIPTION
**Pull request by singhbalwinder**
A fully coupled compset that uses EAMXX as an atmospheric component is added. The compset runs fine for 5 days in full debug mode on Compy.

### CODE REVIEW

File: cime_config/allactive/config_compsets.xml
Comment (singhbalwinder) [inline cime_config/allactive/config_compsets.xml:336]: I chose this name arbitrarily, so I am open to changing it.

Comment (PeterCaldwell) [reply cime_config/allactive/config_compsets.xml:336]: @golaz and @rljacob  - do you have an opinion about how we name coupled compsets using EAMxx?

**Review by PeterCaldwell — DISMISSED** [review]
Looks good to me. Others should review too, and we should get more input on what to name this compset.

**Review by jgfouca — DISMISSED** [review]

**Comment by jgfouca** [comment]
It struck me as a little odd that we have both "eamxx" and "scream" as supported atm component names even though they both refer to the same thing. I suppose we need this for now for backwards compatibility in our test suites.

**Comment by PeterCaldwell** [comment]
good point Jim. I'd prefer to just always call it eamxx and delete the scream option, but maybe that will require a lot of work.

File: cime_config/allactive/config_compsets.xml
Comment (rljacob) [reply cime_config/allactive/config_compsets.xml:336]: We want do distinguish between a WCYCL case with EAM and one with EAMXX.  I think adding "XX" somewhere makes sense.

**Comment by rljacob** [comment]
> good point Jim. I'd prefer to just always call it eamxx and delete the scream option, but maybe that will require a lot of work.

You don't have to change the name of any compset aliases,  just change the longname to use "eamxx" instead of "scream" and delete "scream" from the list of possible atmosphere models.

**Comment by singhbalwinder** [comment]
> You don't have to change the name of any compset aliases, just change the longname to use "eamxx" instead of "scream" and delete "scream" from the list of possible atmosphere models.

That's a great suggestion @rljacob! I can do this in this PR and run all the tests to make sure all are working as expected.

PS. I WIPed this PR.

**Comment by singhbalwinder** [comment]
@PeterCaldwell and I decided to merge this PR (after review) considering that the original objective of executing this compset has already been achieved. That is, this compset can run successfully for several time steps (currently till 538 time steps) in full debug mode on PM-GPU. I will create an issue so that we don't forget about unifying EAMxx and SCREAM in the ATM model definition in various config files.

**Comment by PeterCaldwell** [comment]
Why isn't this merged already? Do we just need someone to manually merge it? @singhbalwinder - if you have a PR which you are waiting for someone to merge, I think it's your job to ask a merger to help you rather than letting the PR side idle for weeks... (which you probably know - I'm just trying to state the obvious in case we were on different pages).

**Comment by singhbalwinder** [comment]
I have added the automerge label but it looks like it needs a fresh review before it can be merged. I should have asked you for a review. Would you please review it so that it is not blocked?

**Review by PeterCaldwell — APPROVED** [review]

**Pull request merged**
